packages/eslint-plugin/test/rules/jsx-import.test.ts

```diff
@@ -73,6 +71,13 @@ ruleTester.run('emotion jsx', rule, {
 
       let ele = <div notCss={{}} />
       `
+    },
+    {
+      code: `
+        /** @jsx jsx */
+        import {jsx} from '@emotion/react'
+        let ele = <div css />
```

Comment:
is this one considered to be valid though? This is equivalent to `css={true}` and that doesn't quite make sense

Reply:
I think you're right, but this PR is meant to be focusing on converting the existing code to TypeScript, so I prefer to do any behaviour changes like this in follow up PRs.

The reason this is here in the first place is because it covers a new condition I have to add in order to make the code valid with TypeScript, since it revealed an unchecked nullish value.

Reply:
So I guess this is sort of valid for this particular rule - although not valid in the general sense. I'm not sure in which rule we should validate this though